src/components/modal-confirm/modal-confirm.jsx

```diff
@@ -41,6 +41,16 @@ var ModalConfirm = React.createClass({
     });
   },
   render: function () {
+    var buttons = this.state.buttons.map((button, i) => {
```

Comment:
Where do these buttons end up coming from? The `dispatcher.fire` in `link-editor.jsx` and such?